prime-angular-frontend/src/app/shared/components/auth/prime-enrolment-access/prime-enrolment-access.component.html

```diff
@@ -98,10 +98,25 @@
     </div>
     <div class="pb-50"
          *ngIf="showHealthAuthorityAccessBtn">
-      <button mat-flat-button
-              color="primary"
-              class="large"
-              (click)="onLogin()">Access Site Registration</button>
+      <div class="mst1">
+        Please enter the provided passcode to continue
+      </div>
+      <form [formGroup]="form"
+            novalidate
+            autocomplete="off">
+        <label>Passcode</label>
+        <mat-form-field class="w-100">
+          <input matInput
```

Comment:
I think Business wanted the input hidden from "shoulder surfers"

Reply:
It is updated.